.github/workflows/release.yml

```diff
@@ -121,6 +128,11 @@ jobs:
           else
             cp "target/${{ matrix.target }}/release/$BIN_NAME" "dist/"
           fi
+
+          if [ "${{ matrix.build }}" = "x86_64-linux" ]; then
+            find target/ -name "*.deb"
+            cp "$(find target/ -name "*.deb")" "dist/"
```

Comment:
This is _odd_.

You can see in [this](https://github.com/PsypherPunk/RustScan/actions/runs/8598981577/job/23560927701) run that the `find` here results in `target/x86_64-unknown-linux-gnu/debian/rustscan_2.1.1-1_amd64.deb`.

However, adding `cp target/x86_64-unknown-linux-gnu/debian/rustscan_*-1_amd64.deb` results in a `cp: cannot stat` error despite the `find` showing that it does exist…?!